pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1263,8 +1263,128 @@ def adjoint(self):
 
     def _controlled(self, wire):
         CSWAP(wires=wire + self.wires)
+        
+class ECR(Operation): 
+    
+    r""" ECR(wires)
+
+    An echoed RZX(pi/2) gate.
+
+    .. math:: \mathtt{ECR} = {1/\sqrt{2}} \begin{bmatrix}
+            0 & 0 & 1 & i \\
+            0 & 0 & i & 1 \\
+            1 & -i & 0 & 0 \\
+            -i & 1 & 0 & 0
+        \end{bmatrix}.
+
+    **Details:**
+
+    * Number of wires: 2
+    * Number of parameters: 0
+
+    Args:
+        wires (int): the subsystem the gate acts on
+        do_queue (bool): Indicates whether the operator should be
+            immediately pushed into the Operator queue (optional)
+        id (str or None): String representing the operation (optional)
+    """
+    
+    
+    num_wires = 2 
+    num_params = 0 
+    def __init__(self, wires, do_queue=True, id=None):
+        super().__init__(wires=wires, do_queue=do_queue, id=id)
```

Comment:
It seems that this definition of `__init__` re-defines the `__init__` of the base class, any reason why it would be good to have it?
```suggestion
```